scripts/test/procedures.py

```diff
@@ -31,24 +47,49 @@ def install_dist():
     subprocess.run(cmd, check=True)
 
 
+def add_last_changes():
+    cmd = ["git", "add", "."]
+    subprocess.run(cmd, check=True)
+
+
+def commit_last_changes():
+    cmd = ["git", "commit", "-am", "\"temp commit\""]
+    subprocess.run(cmd, check=True)
+
+
 def remove_src():
     shutil.rmtree(r"pandas-stubs")
 
 
 def run_mypy_dist():
+    create_mypy_pkg_file()
```

Comment:
When testing the distribution, you do NOT want to have this file present, as the distribution is installed